compose/faq.md

```diff
@@ -111,9 +111,9 @@ github](https://github.com/search?q=in%3Apath+docker-compose.yml+extension%3Ayml
 
```

Comment:
This may be out of scope of this PR, but wondering if it makes sense to move the FAQ topic within Compose docs (Product Manual). It currently appears in the Compose Reference section.